### PR TITLE
CA-270498: Link Help docs to new added USB-passthrough pages.

### PR DESCRIPTION
**Pull request by michael2012z**
This PR implements Context Help for USB-passthrough feature:
USBPage: HelpID = 1020. Show "Tabs" topics. Add "TabPageUSB" in HelpManager.resx.
USBEditPage: HelpID = 6241. Show "Change VM Properties" topic. No change in code.
UsbUsageDialog: Remove help button. 
AttachUsbDialog: Remove help button.

### CODE REVIEW

File: XenAdmin/Help/HelpManager.zh-CN.resx [hunk 1]
```diff
@@ -59,46 +59,46 @@
...
             </xsd:complexType>
           </xsd:element>
           <xsd:element name="resheader">
             <xsd:complexType>
               <xsd:sequence>
-                <xsd:element minOccurs="0" msdata:Ordinal="1" name="value" type="xsd:string"/>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
               </xsd:sequence>
-              <xsd:attribute name="name" type="xsd:string" use="required"/>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
```
Comment (taoyongd) [inline XenAdmin/Help/HelpManager.zh-CN.resx]: Seems above code is not obviously changed . Suggest to keep as it

File: XenAdmin/Help/HelpManager.ja.resx [hunk 2]
```diff
@@ -59,46 +59,46 @@
...
             </xsd:complexType>
           </xsd:element>
           <xsd:element name="resheader">
             <xsd:complexType>
               <xsd:sequence>
-                <xsd:element minOccurs="0" msdata:Ordinal="1" name="value" type="xsd:string"/>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
               </xsd:sequence>
-              <xsd:attribute name="name" type="xsd:string" use="required"/>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
```
Comment (taoyongd) [inline XenAdmin/Help/HelpManager.ja.resx]: above code also

File: XenAdmin/Help/HelpManager.zh-CN.resx (hunk 1 above)
Comment (michael2012z) [reply XenAdmin/Help/HelpManager.zh-CN.resx]: It was updated by Visual Studio automatically, as well as some other meaningless changes in this file. It is not possible to keep them if we use VS.

Comment (MihaelaStoica) [reply XenAdmin/Help/HelpManager.zh-CN.resx]: Yes, Visual Studio does this, but the changes are harmless. 
However, if you want a nice clean diff, then you can discard all these changes and only keep the addition of the three lines at the bottom of this file.

**Pull request merged**